spec/ui_spec.rb

```diff
@@ -79,6 +79,45 @@
     expect(ui.ask_for_notes).to eq("notes")
   end
 
+  it "prints the prompt message for repeating search" do
+    input.string = "\n"
+    ui.ask_search_again
+    expect_msg(UI::SEARCH_AGAIN)
+  end
+
+  it "reads a positive answer to search again" do
+    input.string = "y"
+    expect(ui.ask_search_again).to eq(true)
+  end
+
+  it "reads a negative answer to search again" do
+    input.string = "n"
+    expect(ui.ask_search_again).to eq(false)
+  end
+
+  it "prints the prompt message for a search" do
+    input.string = "\n"
+    ui.ask_for_term
+    expect_msg(UI::SEARCH)
+  end
+
+  it "reads search term" do
+    input.string = "search term"
+    expect(ui.ask_for_term).to eq("search term")
+  end
+
+  it "prints the menu" do
+    menu = [["1", "Option 1"], ["2", "Option 2"]]
```

Comment:
Because I'm a Ruby ignorant (ignorant in general really :stuck_out_tongue_winking_eye: )
I suppose symbols are similar to enums? But I need some string representation in order to print it.